atlasdb-client/src/test/java/com/palantir/atlasdb/table/description/SchemasTest.java

```diff
@@ -53,23 +53,23 @@ public void setup() {
     @Test
     public void testGetFullTableName() {
```

Comment:
Tests need to be renamed, as the tested code no longer gets the full table name.